default/python/AI/AIDependencies.py

```diff
@@ -20,6 +20,8 @@
     my_industry = AIDependencies.INDUSTRY_PER_POP * my_population
 """
 import freeOrionAIInterface as fo  # interface used to interact with FreeOrion AI client  # pylint: disable=import-error
+from common.configure_logging import convenience_function_references_for_logger
+(debug, info, warn, error, fatal) = convenience_function_references_for_logger(__name__)
```

Comment:
Unless I misunderstand what you want.  The indicated line does what you requested.  If you do not want a given logger, then replace it with `_`